src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -22,12 +22,17 @@ interface HelperErrorTextProps {
   id?: string;
   /** Toggles between helper and invalid styling. */
   isInvalid?: boolean;
+  /** Offers the ability to render or not render the content passed in
+   * the `text` prop; `true` by default. */
+  isRenderedText?: boolean;
```

Comment:
Can you give an example of when `text` would be passed, but `isRenderedText` would be set to false?

Reply:
I can't, but that's the way it was set up before along with the `showHelperInvalidText`. Not sure when we would want to hide it but that option is available.